sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/navigation/Navigation.kt

```diff
@@ -0,0 +1,146 @@
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.navArgument
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.presentation.fragment.ColorsFragment
+import uk.gov.hmrc.sample_compose_fragments.presentation.fragment.FragmentContainer
+import uk.gov.hmrc.sample_compose_fragments.presentation.navigation.NavigationConstants.NAV_ARG_TOP_APP_BAR_TITLE
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.ComponentListScreen
+
+object NavigationConstants {
+    const val NAV_ARG_TOP_APP_BAR_TITLE = "NAV_ARG_TOP_APP_BAR_TITLE"
+}
+
+sealed class NavigationScreen(val route: String, @StringRes val resourceId: Int, @DrawableRes val icon: Int) {
+    object Organisms : NavigationScreen("organisms", R.string.title_organisms, R.drawable.ic_organisms)
+    object Molecules : NavigationScreen("molecules", R.string.title_molecules, R.drawable.ic_molecules)
+    object Atoms : NavigationScreen("atoms", R.string.title_atoms, R.drawable.ic_atoms)
+    object Colors : NavigationScreen("colors", R.string.title_colors, R.drawable.ic_colors)
+}
+
+@Composable
+fun ComponentsBottomNavigation(navController: NavHostController) {
+    NavigationBar(
+        modifier = Modifier.shadow(4.dp),
+        containerColor = HmrcTheme.colors.hmrcWhiteBackground
+    ) {
+        val navBackStackEntry by navController.currentBackStackEntryAsState()
+        val currentDestination = navBackStackEntry?.destination
+
+        val navigationItems = listOf(
+            NavigationScreen.Organisms,
+            NavigationScreen.Molecules,
+            NavigationScreen.Atoms,
+            NavigationScreen.Colors
+        )
+
+        navigationItems.forEach { screen ->
+            val selected = currentDestination?.hierarchy?.any { it.route == screen.route } == true
+            NavigationBarItem(
+                icon = {
+                    Icon(
+                        tint = if (selected) HmrcTheme.colors.hmrcBlue else HmrcTheme.colors.hmrcGrey1,
+                        painter = painterResource(id = screen.icon),
+                        contentDescription = null
+                    )
+                },
+                label = { Text(stringResource(screen.resourceId)) },
+                selected = currentDestination?.hierarchy?.any { it.route == screen.route } == true,
+                onClick = {
+                    navController.navigate(screen.route) {
+                        popUpTo(navController.graph.findStartDestination().id) { saveState = true }
+                        launchSingleTop = true
+                        restoreState = true
+                    }
+                }
+            )
+        }
+    }
+}
+
+@Composable
+fun ComponentsNavGraph(
```

Comment:
Are we able to try and use the xml nav graphs rather than this compose one? It will be a more similar example to our existing main app to practice integration without needing so much upfront refactor to start using compose (hopefully!).